<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recDialog</class>
 <widget class="QDialog" name="recDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>校名:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="institutionlineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>院系:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="departmentlineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>专业:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="majorlineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>网址:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="weblineEdit_3"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>地址:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addrlineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="TypecomboBox">
         <item>
          <property name="text">
           <string>类型</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>综合</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>理工</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>工业</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>科技</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>师范</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>农业</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>财经</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>航空</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>交通</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>邮电</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>其他</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="areacomboBox">
         <property name="minimumSize">
          <size>
           <width>133</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>区域</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>华东</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>华中</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>华南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>华北</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>东北</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>西北</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>西南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>自治区</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>行政特区</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="provincecomboBox">
         <property name="minimumSize">
          <size>
           <width>133</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>北京</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>简介:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="introductiontextEdit">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2700</width>
             <height>800</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>补充:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="addtionaltextEdit">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2700</width>
             <height>800</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>681</width>
       <height>321</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6810</width>
       <height>3210</height>
      </size>
     </property>
     <row>
      <property name="text">
       <string>01</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>02</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>03</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>04</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>05</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>06</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>07</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>08</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>09</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>19</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>20</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>联系人</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>职位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系电话</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>邮箱</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>自定义1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>自定义2</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <zorder>tableWidget</zorder>
  <zorder>label</zorder>
  <zorder>institutionlineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>departmentlineEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>weblineEdit_3</zorder>
  <zorder>label_4</zorder>
  <zorder>majorlineEdit</zorder>
  <zorder>addtionaltextEdit</zorder>
  <zorder>introductiontextEdit</zorder>
  <zorder>addrlineEdit</zorder>
  <zorder>label_6</zorder>
  <zorder>provincecomboBox</zorder>
  <zorder>areacomboBox</zorder>
  <zorder>label_5</zorder>
  <zorder>label_7</zorder>
  <zorder>introductiontextEdit</zorder>
  <zorder>TypecomboBox</zorder>
  <zorder>addBtn</zorder>
  <zorder>cancelBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
